src/main/java/uk/gov/hmcts/reform/professionalapi/util/RefDataUtil.java

```diff
@@ -273,7 +273,11 @@ public static Pageable createPageableObject(Integer page, Integer size, Sort sor
     }
 
     public static String getShowDeletedValue(String showDeleted) {
-        return "True".equalsIgnoreCase(showDeleted) ? ProfessionalApiConstants.TRUE : ProfessionalApiConstants.FALSE;
+        return ProfessionalApiConstants.TRUE.equalsIgnoreCase(showDeleted) ? ProfessionalApiConstants.TRUE : ProfessionalApiConstants.FALSE;
+    }
+

```

Comment:
can you one generic method here instead of two ..both are doing same

Reply:
One is checking if String equals to `True`, other is checking if string equals to `False` - they also return different things on the else condition - would this not cause issues if I create one generic method?